Shared/Classes/Utils/Services/OTDeepLinkService.m

```diff
@@ -146,10 +146,10 @@ - (void)handleDeepLinkWithKey:(NSString *)key pathComponents:(NSArray *)pathComp
         [mainViewController switchToGuide];
     }
     else if ([key isEqualToString:@"tutorial"]) {
-        OTMainViewController *mainViewController = [self popToMainViewController];
-        UIStoryboard *tutorialStoryboard = [UIStoryboard storyboardWithName:@"Tutorial" bundle:nil];
-        UIViewController *tutorialController = [tutorialStoryboard instantiateInitialViewController];
-        [mainViewController presentViewController:tutorialController animated:YES completion:nil];
+//        OTMainViewController *mainViewController = [self popToMainViewController];
```

Comment:
@Veronica-Gliga : Please create a task for updating this sequence with a method in OTAppState class. There should be a method to automatically select the Menu Tab in this case.